TDD_Training/TDD_Training/Interface/Multiples.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TDD_Training.Interface
+{
+    public interface Multiples
```

Comment:
Is this being used?

Reply:
No. I thought of removing this before committing. But I forgot to remove it. I will update the code.